files/usr/share/defaults/jeos-firstboot.conf

```diff
@@ -16,3 +16,9 @@
 # initial password for the root user will be skipped. In this case is
 # expected that the root password was set by other means.
 # JEOS_PASSWORD_ALREADY_SET='yes'
+
+# If set to a nonempty value, the dialog boc for accepting the EULA
+# will be skipped. Use only this option when building images (ISO /
+# PXE / OEM) that are part of another product and than can be used
+# when this license was accepted.
+# JEOS_SKIP_EULA='yes'
```

Comment:
It should be more clear that by setting this option the EULA is agreed to - so maybe `JEOS_EULA_ACCEPTED=yes`?

Reply:
Indeed, I renamed it to `JEOS_EULA_ALREADY_AGREED`, to be similar to the password one.